### PR TITLE
New GUI

### PR DESCRIPTION
**Pull request by firenoo**
A new GUI. Features:

- New Icons provided via AE2
- Resizable GUI and now respects screen resize
- Removed select button in favor of using the actual entry as a select button
- Name moved to top, type 2nd, Frequency 3rd, channels used 4th
- Hover over icon for position details

### CODE REVIEW

File: src/main/java/com/projecturanus/betterp2p/client/gui/widget/WidgetButton.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    abstract fun mousePressed(mouseX: Int, mouseY: Int)
+
+    fun draw(mc: Minecraft?, mouseX: Int, mouseY: Int, partialTicks: Float) {
+        val tessellator = Tessellator.instance
+        gui.bindTexture(gui.BACKGROUND)
+        GL11.glColor4f(1.0f, 1.0f, 1.0f, 1.0f)
+        field_146123_n = mouseX >= xPosition && mouseY >= yPosition && mouseX < xPosition + width && mouseY < yPosition + height
+        val k = getHoverState(field_146123_n)
+        GL11.glEnable(GL11.GL_BLEND)
+        OpenGlHelper.glBlendFunc(770, 771, 1, 0)
```
Comment (eigenraven) [inline src/main/java/com/projecturanus/betterp2p/client/gui/widget/WidgetButton.kt:37]: 770 = GL11.GL_SRC_ALPHA
771 = GL11.GL_ONE_MINUS_SRC_ALPHA
iirc but double check, the line below is not needed either

File: src/main/java/com/projecturanus/betterp2p/client/gui/GuiAdvancedMemoryCard.kt [hunk 2]
```diff
@@ -1,54 +1,68 @@
...
-    private val guiTop: Int by lazy { (height - this.ySize) / 2 }
+    private lateinit var _guiLeft: Lazy<Int>
+    private var guiLeft: Int
+        get() = _guiLeft.value
+        set(value) {
+            _guiLeft = lazyOf(value)
+        }
+
+    private lateinit var _guiTop: Lazy<Int>
+    private var guiTop: Int
```
Comment (eigenraven) [inline src/main/java/com/projecturanus/betterp2p/client/gui/GuiAdvancedMemoryCard.kt:33]: why are these necessary and you don't just use `this.guiTop` from the superclass?

Comment (firenoo) [reply src/main/java/com/projecturanus/betterp2p/client/gui/GuiAdvancedMemoryCard.kt:33]: Those fields don't exist.

File: src/main/java/com/projecturanus/betterp2p/client/gui/widget/WidgetButton.kt (hunk 1 above)
Comment (firenoo) [reply src/main/java/com/projecturanus/betterp2p/client/gui/widget/WidgetButton.kt]: Fixed.

File: src/main/java/com/projecturanus/betterp2p/client/gui/GuiAdvancedMemoryCard.kt (hunk 2 above)
Comment (eigenraven) [reply src/main/java/com/projecturanus/betterp2p/client/gui/GuiAdvancedMemoryCard.kt:33]: Ah right those are in GuiContainer, not GuiScreen

**Review by eigenraven — APPROVED** [review]
